municipal_finance/templates/docs.html

```diff
@@ -34,6 +34,7 @@
 
 <div class="page-wrapper">
   <div class="content">
+    {% for notice in site_notices %}<h3>{{ notice.content | safe }}</h3>{% endfor %}
```

Comment:
Let's not use a h3 for notices as they are not structurally headings. A p with a class should suffice e.g. `<p class="site-notice">....</p>`

Reply:
Same here, changed to div with it's own CSS